release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2872,6 +2872,42 @@ This section will continue to be updated over time to provide notes on enhanceme
+* Previously, you could not use all available machine types in a zone because all zones in a region were assumed to have the same set of machine types. With this release, all machine types are available in all enabled zones. (link:https://issues.redhat.com/browse/OCPBUGS-46432[*OCPBUGS-46432*])
+* Previously, the installation program was not compliant with PCI-DSS/BAFIN regulations. With this release, the cross-tenant replication in {azure-first} is disabled, which reduces the chance of unauthorized data access and ensures strict adherence to data governance policies. (link:https://issues.redhat.com/browse/OCPBUGS-48119[*OCPBUGS-48119*])
+* Previously, when you clicked the *Don't show again* link in the Red Hat Ansible Lightspeed modal, it did not display the correct *General User Preference* tab when one of the other *User Preference* tabs was open. With this release, clicking the *Don't show again* link goes to the correct *General User Preference* tab. (link:https://issues.redhat.com/browse/OCPBUGS-48227[*OCPBUGS-48227*])
+* Previously, when a Google Cloud Platform (GCP) service account was created, the account would not always be immediately available. When the account was not available for updates, the installation program received failures when adding permissions to the account. According to Google, https://cloud.google.com/iam/docs/retry-strategy, a service account may be created, but is not active for up to 60 seconds. With this release, the service account is updated on an exponential backoff to give the account enough time to update correctly. (link:https://issues.redhat.com/browse/OCPBUGS-48359[*OCPBUGS-48359*])
+* Previously, on RHEL9 FIPS STIG compliant machines, The SHA-1 key caused the release signature verification to fail because of the restriction to use weak keys.
+With this release, the key used by oc-mirror for release signature verification is changed and release images are signed by a new SHA256 trusted-key that is different from the old SHA-1 key. (link:https://issues.redhat.com/browse/OCPBUGS-48363[*OCPBUGS-48363*])
+* Previously, the {olm-first} would sometimes concurrently resolve the same namespace in a cluster. This led to subscriptions reaching a terminal state of `ConstraintsNotSatisfiable`, because two concurrent processes interacted with a subscription and this caused a CSV file to become unassociated. With this release, {olm} can resolve concurrent namespaces for a subscription so no CSV remains unassociated. (link:https://issues.redhat.com/browse/OCPBUGS-45845[*OCPBUGS-45845*])
+* Previously, creating a hosted control planes (HCP) cluster using the HCP cli failed on the release image check to verify that the release image was a multi-arch image. With this release, the HCP cli does not fail to create a hosted cluster because the updated cli code with the multi-arch release image check is available. (link:https://issues.redhat.com/browse/OCPBUGS-44927[*OCPBUGS-44927*])
```

Comment:
```suggestion
* Previously, creating a hosted control planes (HCP) cluster using the HCP CLI failed on the release image check to verify that the release image was a multi-arch image. With this release, the HCP CLI does not fail to create a hosted cluster because the updated CLI code with the multi-arch release image check is available. (link:https://issues.redhat.com/browse/OCPBUGS-44927[*OCPBUGS-44927*])
```